cmd/sonobuoy/app/run.go

```diff
@@ -71,6 +75,15 @@ func NewCmdRun() *cobra.Command {
 }
 
 func submitSonobuoyRun(cmd *cobra.Command, args []string) {
+
+	var s *spinner.Spinner
+
+	if (runflags.wait > 0 && runflags.waitOutput == "spinner") {
```

Comment:
I'd modify this and the flag definition in a few ways:
 - const values like `spinner`
 - create a type with custom `Set` method so that pflag calls it when set. Validate the user provided only one of the allowable values. See `AddRBACModeFlags` for an example.